Dockerfile

```diff
@@ -108,11 +108,13 @@ ENV RUSTUP_HOME=/usr/local/rustup \
 RUN apk add --no-cache --virtual .build-deps \
         build-base \
         python3-dev \
+        gcc \
```

Comment:
No need for these two, since `build-base` is already a meta package that includes them both .

Reply:
Removed. Made some tests with base image and just copied those directly from those tries and/or cryptography installation guide.